src/applications/pensions/config/chapters/02-military-history/servicePeriod.js

```diff
@@ -74,7 +73,7 @@ export default {
     type: 'object',
     required: ['serviceBranch', 'activeServiceDateRange'],
     properties: {
-      serviceBranch: checkboxGroupSchema(Object.keys(serviceBranchLabels)),
+      serviceBranch,
```

Comment:
It might make sense to start with one PR that just does yesNoSchema, then iteratively follow up with additional PRs for checkboxGroupSchema, radioSchema, etc, to keep the code changes smaller and reduce the chance of errors